### PR TITLE
feat(meta cache): stats meta cache unhit in prometheus

### PR DESCRIPTION
**Pull request by soundOfDestiny**
I hereby agree to the terms of the [RisingWave Labs, Inc. Contributor License Agreement](https://gist.github.com/TennyZhuang/f00be7f16996ea48effb049aa7be4d66#file-rw_cla).

## What's changed and what's your intention?

stats meta cache unhit in prometheus

## Checklist For Contributors

- [x] I have written necessary rustdoc comments
- [x] I have demonstrated that backward compatibility is not broken by breaking changes and created issues to track deprecated features to be removed in the future. (Please refer to the issue)
- [x] All checks passed in `./risedev check` (or alias, `./risedev c`)

## Checklist For Reviewers

- [ ] I have requested macro/micro-benchmarks as this PR can affect performance substantially, and the results are shown.

## Documentation

- [x] My PR **DOES NOT** contain user-facing changes.

<details><summary>Click here for Documentation</summary>

### Types of user-facing changes

Please keep the types that apply to your changes, and remove the others.

- Installation and deployment
- Connector (sources & sinks)
- SQL commands, functions, and operators
- RisingWave cluster configuration changes
- Other (please specify in the release note below)

### Release note

</details>

### CODE REVIEW

File: src/storage/src/hummock/store/version.rs [hunk 1]
```diff
@@ -409,6 +409,8 @@ impl HummockVersionReader {
     }
 }
 
+const SLOW_ITER_FETCH_META_DURATION: f64 = 5.0;
```
Comment (Li0k) [inline src/storage/src/hummock/store/version.rs:412]:
```rust
SLOW_ITER_FETCH_META_DURATION_SECOND
```

File: src/storage/src/hummock/store/version.rs [hunk 2]
```diff
@@ -653,12 +655,24 @@ impl HummockVersionReader {
...
+            local_cache_meta_block_unhit += resp.2;
             flatten_resps[req_count - req_index - 1] = Some(resp);
         }
-        timer.observe_duration();
+        let fetch_meta_duration_sec = timer.stop_and_record();
+        self.state_store_metrics
+            .iter_fetch_meta_cache_unhits
+            .with_label_values(&[table_id_label])
+            .observe(local_cache_meta_block_unhit as f64);
+        if fetch_meta_duration_sec > SLOW_ITER_FETCH_META_DURATION {
```
Comment (Li0k) [inline src/storage/src/hummock/store/version.rs:670]: Shall we need the debug log to quickly locate which table (epoch...) this query appears in?

**Review by Li0k — APPROVED** [review]
Rest LGTM

File: src/storage/src/hummock/store/version.rs (hunk 1 above)
Comment (soundOfDestiny) [reply src/storage/src/hummock/store/version.rs]: > ```rust
> SLOW_ITER_FETCH_META_DURATION_SECOND
> ```

good catch. fixed

File: src/storage/src/hummock/store/version.rs (hunk 2 above)
Comment (soundOfDestiny) [reply src/storage/src/hummock/store/version.rs]: > Shall we need the debug log to quickly locate which table (epoch...) this query appears in?

I have just added some debug log. thanks

**Pull request merged**